packages/kg-default-nodes/lib/nodes/header/HeaderRenderer.js

```diff
@@ -0,0 +1,71 @@
+import {addCreateDocumentOption} from '../../utils/add-create-document-option';
+
+function slugify(str) {
+    // Remove any non-word character with whitespace
+    str = str.replace(/[^\w\s]/gi, '');
+    
+    // Replace any whitespace character with a dash
+    str = str.replace(/\s+/g, '-');
+    
+    // Convert to lowercase
+    str = str.toLowerCase();
+    
+    return str;
+}
```

Comment:
Could we use
```
const {slugify} = require('@tryghost/kg-utils');
```
I think the implementation is different, so could maybe cause issues with the ids that change if we migrate over from mobiledoc to lexical